packages/common/src/types/dbtCloud.ts

```diff
@@ -7,15 +7,33 @@ export type DbtCloudMetric = {
     label: string;
 };
 
+/**
+ * Response from dbt cloud metadata api containing a list of metric definitions
+ */
```

Comment:
should this file be inside types/api?

Reply:
This is the api response we get _from dbt Cloud_ when we make a request from our backend to their API. So it's separate from the payload we return to our FE from the backend API